backend/src/govsg/models/govsg-op.ts

```diff
@@ -1,5 +1,69 @@
-import { Model, Table } from 'sequelize-typescript'
-import { GovsgMessage } from './govsg-message'
+import {
+  BelongsTo,
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  Table,
+} from 'sequelize-typescript'
+import { Campaign } from '@core/models'
+import { GovsgMessageStatus } from '@core/constants'
 
 @Table({ tableName: 'govsg_ops', underscored: true, timestamps: true })
-export class GovsgOp extends Model<GovsgMessage> {}
+export class GovsgOp extends Model<GovsgOp> {
+  @ForeignKey(() => Campaign)
```

Comment:
identical to `GovsgMessage` — need to duplicate code for Sequelize ORM methods to work. Sequelize sucks.

Reply:
wdym ah you want to commit directly?